### ISSUE TITLE
请问可以提供数据集以及具体的运行复现手册吗？

### ISSUE THREAD

**Issue by yuwxyun275** [Coco-Hut/SE-HSSL#1]
阅读完您的工作之后，我认为你们的工作很棒！对领域贡献非常大，请问是否可以提供相应的数据集以及如何运行这些代码？

**Comment by Coco-Hut** [comment]
Hi, thank you for your interest in our paper! We conduct our experiments using the datasets from the TriCL paper. Please download datasets.zip from the [TriCL repository](https://github.com/wooner49/TriCL). Once you have the dataset folder, you can run main.py to perform the experiments. If you want to use the membership-level loss, please first call the function “generate_sample“ to obtain the positive and negative pairs.

**Comment by yuwxyun275** [comment]
I really appreciate how quickly you respond and help me! However, after running with the original YAML parameters, I didn't achieve the results from the paper. For example, in Cora and Citeseer, I am 0.7% and 1% off, respectively. I think this might be due to hyperparameters, and I was wondering if you could provide the best hyperparameters from the paper

![Image](https://github.com/user-attachments/assets/544f0147-e46c-4dcf-b5d2-f6a5457d49a6)

**Comment by Coco-Hut** [comment]
Hi, I apologize for not being able to provide the optimal parameter combination file, as I no longer have access to the server where I conducted the experiments last year. Based on my experience, the parameters that most significantly affect the results include: w_g, w_m, n_epoch, lr, lambda_n, lambda_g, and weight decay. Sometimes, even small changes in these parameters can lead to significant fluctuations in the results.

**Comment by yuwxyun275** [comment]
I appreciate your response. Could you please remind me of the range within which these parameters should be searched?

**Comment by Coco-Hut** [comment]
I remember that the epochs are tuned within the range of 250 to 330. However, it is hard for me to remember other important hyperparameters with small values. When tuning the parameters, please use a smaller step size, as the performance in the figure is quite close. And weight decay can sometimes significantly influence the accuracy. I will take some time to search for the optimal hyperparameters again once I complete my current work.

**Comment by yuwxyun275** [comment]
Thank you very much for paying such great attention to the questions I raised. I look forward to receiving your reply soon!
